tests/integration_tests/instances.py

```diff
@@ -60,7 +59,7 @@ class IntegrationInstance:
     def __init__(
         self,
         cloud: "IntegrationCloud",
-        instance: BaseInstance,
+        instance,
```

Comment:
This was fairly useful when it comes to editor autocomplete. Why does mypy not like it?